src/Components/Article/ArticleProducts.js

```diff
@@ -0,0 +1,23 @@
+import Button from "../Button/Button";
+
+
+function ArticleProducts({redLabelText, imgTas, imgAlt ,nameBag, Price, Children, text}){
+    function showAlert() {
+        alert(text);
+    }
+
+    return(
+        <article>
+            <span>{redLabelText}</span>
+            <img src={imgTas} alt={imgAlt}/>
+            <p> {nameBag} </p>
+            <h4> {Price} </h4>
```

Comment:
Bij het Product Component is het wenselijk dat de €
en ,- van de prijs worden toegevoegd in het 
component zelf, zodat we altijd alleen een cijfer mee 
hoeven geven :)